lib/Button/BackButtonMobile.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import ChevronLeftWhiteIcon from 'lib/assets/icon-chevron-left-white.svg';
+
+export const BackButtonMobile = styled.button`
+  @media (min-width: ${({ theme }) => theme.breakpoints.mobile}) {
+    display: none;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobile}) {
```

Comment:
Is there a need to keep this rule? You could leave the mobile styles as default.

Reply:
Because I have issues :P: **that's** for desktop **and that's** for mobile.

If you want to refactor it, go for it.

Reply:
I'll probably do some refactors eventually because as it's, the layout breaks when the resolution is exactly equal to the mobile breakpoint :joy: